demo/package.json

```diff
@@ -24,6 +27,10 @@
   "dependencies": {
     "d3-scale": "^1.0.7",
     "d3-selection": "^1.2.0",
-    "preact-render-to-string": "^4.1.0"
+    "mobx": "^5.9.0",
+    "mobx-preact": "^3.0.0",
+    "mobx-state-tree": "^3.12.2",
+    "preact-render-to-string": "^4.1.0",
```

Comment:
We have a `next` tag for `preact-render-to-string` which points to 5.x and supports ceviche :+1: